File: ABC/401-450/404/a.py
def main():
  s = input()
  cnt = [0]*26
  for i in range(len(s)):
    cnt[ord(s[i])-ord('a')] += 1
  
  for i in range(26):
    if cnt[i] == 0:
      print(chr(i+ord('a')))
      exit()

if __name__ == '__main__':
  main()

File: ABC/401-450/404/a.cpp
#include <iostream>
#include<vector>
using namespace std;

int main(){
  vector<int> cnt(26);
  string s;
  cin >> s;
  for(int i=0;i<s.size();i++) cnt[s[i]-'a']++;
  for(int i=0;i<26;i++){
    if(cnt[i] == 0){
      cout << char('a'+i) << endl;
      return 0;
    }
  }
  return 0;
}

